music21/romanText/tsvConverter.py

```diff
@@ -717,18 +779,26 @@ def _m21ToTsv_v1(self):
             thisEntry.measure = thisRN.measureNumber
             thisEntry.beat = thisRN.beat
             thisEntry.totbeat = None
-            thisEntry.timesig = thisRN.getContextByClass(meter.TimeSignature).ratioString
-            thisEntry.op = self.m21Stream.metadata.opusNumber
-            thisEntry.no = self.m21Stream.metadata.number
-            thisEntry.mov = self.m21Stream.metadata.movementNumber
+            ts = thisRN.getContextByClass(meter.TimeSignature)
+            if ts is None:
+                thisEntry.timesig = ''
+            else:
+                thisEntry.timesig = ts.ratioString
+            thisEntry.extra["op"] = self.m21Stream.metadata.opusNumber
+            thisEntry.extra["no"] = self.m21Stream.metadata.number
+            thisEntry.extra["mov"] = self.m21Stream.metadata.movementNumber
```

Comment:
single quotes.  and note that these might be None -- so better to add ` or ''` afterwards.  mypy did not catch this because `extra` is not typed.